gslib/utils/text_util.py

```diff
@@ -264,68 +264,68 @@ def PrintableStr(input_val):
+  This function writes text to a file descriptor as the
+  builtin print function would, favoring unicode encoding.
 
-    Aguments and return values are the same as documented in
-    the Python 2 print function.
+  Aguments and return values are the same as documented in
+  the Python 2 print function.
+  """
+  def _get_args(**kwargs):
+    """Validates keyword arguments that would be used in Print
+
+    Valid keyword arguments, mirroring print(), are 'sep',
+    'end', and 'file'. These must be of types string, string,
+    and file / file interface respectively.
+
+    Returns the above kwargs of the above types.
     """
-    def _get_args(**kwargs):
-        """Validates keyword arguments that would be used in Print
-
-        Valid keyword arguments, mirroring print(), are 'sep',
-        'end', and 'file'. These must be of types string, string,
-        and file / file interface respectively.
-
-        Returns the above kwargs of the above types.
-        """
-        expected_keywords = collections.OrderedDict([
-            ('sep', ' '),
-            ('end', '\n'),
-            ('file', sys.stdout)])
-
-        for key, value in kwargs.items():
-            if key not in expected_keywords:
-                error_msg = (
-                    '{} is not a valid keyword argument. '
-                    'Please use one of: {}')
-                raise KeyError(
-                    error_msg.format(
-                        key,
-                        ' '.join(expected_keywords.keys())))
-            else:
-                expected_keywords[key] = value
-
-        return expected_keywords.values()
-
-    sep, end, file = _get_args(**kwargs)
-
-    sep = six.ensure_binary(sep)
-    end = six.ensure_binary(end)
-    byte_objects = [six.ensure_binary(item) for item in objects]
-    data = sep.join(byte_objects)
-    data += end
-    write_to_fd(file, data)
+    expected_keywords = collections.OrderedDict([
+      ('sep', ' '),
+      ('end', '\n'),
+      ('file', sys.stdout)])
+
+    for key, value in kwargs.items():
+      if key not in expected_keywords:
+        error_msg = (
+          '{} is not a valid keyword argument. '
+          'Please use one of: {}')
+        raise KeyError(
+          error_msg.format(
+            key,
+            ' '.join(expected_keywords.keys())))
+        else:
+          expected_keywords[key] = value
+
+    return expected_keywords.values()
+
+  sep, end, file = _get_args(**kwargs)
+
+  sep = six.ensure_binary(sep)
+  end = six.ensure_binary(end)
+  byte_objects = [six.ensure_binary(item) for item in objects]
+  data = sep.join(byte_objects)
+  data += end
+  write_to_fd(file, data)
 
 
 def write_to_fd(fd, data):
-    """Write given data to given file descriptor, doing any conversions needed.isinstance"""
-    if six.PY2:
+  """Write given data to given file descriptor, doing any conversions needed.isinstance"""
+  if six.PY2:
+    fd.write(data)
+  else:
```

Comment:
The meat of this function is indented within an else block.  Since we don't do anything after the if/else block, I'd Instead prefer returning at the end of the if block, e.g.:

```
  if six.PY2:
    fd.write(data)
    return

  # Otherwise, PY3:
  if isinstance(data, bytes):
    ...
```

Reply:
Oh yeah, that would be more clean! Will refactor real quick